### DISCUSSION TITLE
Nueva convocatoria de Launch X Latam

### DISCUSSION THREAD

**Discussion by T0ny-dev** [Launch-X-Latam/MisionFrontEnd#230] [General]
Buen día comunidad, quería saber si va haber una nueva convocatoria de launch-X-Latam próxima a futuro.Porque un amigo esta interesado y me pregunto, alguno tiene información de las próximas fechas o convocatorias.

**Comment by doguedogue** [comment]
https://twitter.com/innovaccionv?lang=es

**Reply by T0ny-dev** [reply]
Muchas gracias me sirvió bastante y pude pasar la información ingeniero. 🚀

**Comment by T0ny-dev** [comment]
> https://twitter.com/innovaccionv?lang=es

Muy buena
